keps/sig-scheduling/1923-try-nominated-node-first/kep.yaml

```diff
@@ -0,0 +1,26 @@
+title: Try Nominated Node First
+kep-number: 1923
+authors:
+  - "@chendave"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-scheduling
+status: implementable
+creation-date: 2020-09-29
+reviewers:
+  - "@sig-scheduling-leads"
+approvers:
+  - "@sig-scheduling-leads"
+see-also:
+  - "/keps/sig-scheduling/1923-try-nominated-node-first"
+stage: alpha
```

Comment:
I think we could target Beta directly, but that's a question for approvers.

You would need to complete the Production Readiness questionnaire in that case.